ob-elasticsearch.el

```diff
@@ -49,13 +49,21 @@ block.")
 (defvar org-babel-tangle-lang-exts)
 (add-to-list 'org-babel-tangle-lang-exts '("es" . "es"))
 
+(defun es-org-aget (key alist)
+  (assoc-default (intern key) alist))
+
 (defun org-babel-expand-body:es (body params)
   "This command is used by org-tangle to create a file with the
 source code of the elasticsearch block. If :tangle specifies a
 file with the .sh extension a curl-request is created instead of
 just a normal .es file that contains the body of the block.."
   (let ((ext (file-name-extension
-              (cdr (assoc :tangle params)))))
+              (cdr (assoc :tangle params))))
+        (body (s-format body 'es-org-aget
```

Comment:
This adds a requirement for s.el, so can you add it to the package dependencies and add `require` for it?

Reply:
Whoops! I forgot that I already have `s.el` a requirement for es-mode, it still needs to require though